install/kubernetes/helm/istio/test-values/values-e2e.yaml

```diff
@@ -16,7 +16,8 @@ global:
     mode: REGISTRY_ONLY
 
 prometheus:
-  scrapeInterval: 5s
+  scrapeInterval: 20s
+  tag: v2.12.0
```

Comment:
We upgrade in the default case now, probably do not need to hardcode here

Reply:
checking...